PrismSampleApp/PrismSampleApp/CustomRender/CustomRenderer.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Xamarin.Forms;
+
+namespace PrismSampleApp.CustomRender
+{
+    public class CustomRenderer : Entry
```

Comment:
Move this class file under CustomControl and rename it as CustomEntry.

Reply:
moved and renamed